src/workflows-app/SubmissionConfig.test.js

```diff
@@ -419,30 +428,1484 @@ describe('BaseSubmissionConfig with workflowsAppStore', () => {
+        WorkflowScript: {
+          get: mockWdlResponse,
+        },
+      };
+    });
+
+    // ** ACT **
+    await act(async () =>
+      render(
+        h(BaseSubmissionConfig, {
+          methodId: '123',
+          name: 'test-azure-ws-name',
+          namespace: 'test-azure-ws-namespace',
+          workspace: mockAzureWorkspace,
+        })
+      )
+    );
+
+    // ** ASSERT **
+    expect(mockRunSetResponse).toHaveBeenCalledTimes(1);
+    expect(mockTypesResponse).toHaveBeenCalledTimes(1);
+    expect(mockMethodsResponse).toHaveBeenCalledTimes(1);
+    expect(mockSearchResponse).toHaveBeenCalledTimes(1);
+    expect(mockWdlResponse).toHaveBeenCalledTimes(1);
+
+    const table = screen.getByRole('table');
+
+    const rows = within(table).queryAllByRole('row');
+    expect(rows.length).toBe(5);
+
+    const headers = within(rows[0]).queryAllByRole('columnheader');
+    expect(headers.length).toBe(4);
+
+    const cells = within(rows[1]).queryAllByRole('cell');
+    expect(cells.length).toBe(4);
+  });
+
+  it('should initially populate the inputs definition table with attributes determined by the previously executed run set', async () => {
+    // ** ARRANGE **
+    const user = userEvent.setup();
+    const mockRunSetResponse = jest.fn(() => Promise.resolve(runSetResponse));
+    const mockMethodsResponse = jest.fn(() => Promise.resolve(methodsResponse));
+    const mockSearchResponse = jest.fn((_root, _instanceId, recordType) => Promise.resolve(searchResponses[recordType]));
+    const mockTypesResponse = jest.fn(() => Promise.resolve(typesResponse));
+    const mockWdlResponse = jest.fn(() => Promise.resolve('mock wdl response'));
+
+    Ajax.mockImplementation(() => {
+      return {
+        Cbas: {
+          runSets: {
+            getForMethod: mockRunSetResponse,
+          },
+          methods: {
+            getById: mockMethodsResponse,
+          },
+        },
+        WorkspaceData: {
+          queryRecords: mockSearchResponse,
+          describeAllRecordTypes: mockTypesResponse,
+        },
+        WorkflowScript: {
+          get: mockWdlResponse,
+        },
+      };
+    });
+
+    // ** ACT **
+    await act(async () =>
+      render(
+        h(BaseSubmissionConfig, {
+          methodId: '123',
+          name: 'test-azure-ws-name',
+          namespace: 'test-azure-ws-namespace',
+          workspace: mockAzureWorkspace,
+        })
+      )
+    );
+
+    // ** ASSERT **
+    expect(mockRunSetResponse).toHaveBeenCalledTimes(1);
+    expect(mockTypesResponse).toHaveBeenCalledTimes(1);
+    expect(mockMethodsResponse).toHaveBeenCalledTimes(1);
+    expect(mockSearchResponse).toHaveBeenCalledTimes(1);
+    expect(mockWdlResponse).toHaveBeenCalledTimes(1);
+
+    const button = screen.getByRole('button', { name: 'Inputs' });
+
+    // ** ACT **
+    await user.click(button);
+
+    // ** ASSERT **
+    const table = screen.getByRole('table');
+    const rows = within(table).queryAllByRole('row');
```

Comment:
Testing Library recommends `getBy...` over `queryBy...` for everything except checking if an element does _not_ exist.
https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-query-variants-for-anything-except-checking-for-non-existence